machine.md

```diff
@@ -4,6 +4,41 @@ Finetuning different aspects of the hardware which are not device related
 (BIOS, mainboard, ...) is sometimes necessary to allow Guest Operating Systems
 to properly boot and reboot.
 
+## Machine Type
+
+QEMU is able to work with two different classes of chipsets for x86\_64, so
+called machine types. The x86\_64 chipsets are i440fx (also called pc) and q35.
+They are versioned based on qemu-system-$ARCH, following the format
+`pc-${machine_type}-${qemu_version}`, e.g. `pc-i440fx-2.10` and `pc-q35-2.10`.
+
+KubeVirt defaults to QEMU's newest q35 machine type. If custom machine type is
+desired, it is configurable via following structure:
+
+```
+metadata:
+  name: myvm
+spec:
+  domain:
+    machine:
+      # This value indicates QEMU machine type.
+      type: pc-q35-2.10
+    resources:
+      requests:
+        memory: 512M
+    devices:
+      disks:
+      - name: myimage
+        volumeName: myimage
+        disk: {}
+  volumes:
+    - name: myimage
+      persistentVolumeClaim:
+        claimName: myclaim
+```
+
+Comparision of the machine types' internals can be found
```

Comment:
nit: `types'` is the plural possessive form. I think `type's` would be correct here.

Reply:
(just making sure) isn't plural possessive the correct choice here? It is link to docs about the internals of 2 distinct machine types.

Reply:
You are correct! I had to reread the statement a few times to understand your question. I had read "machine" as the object of the preposition, but as you pointed out, the comparison is between the internals of multiple types.